packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types'
 import themed from 'react-themed'
 import classnames from 'classnames'
 
-@themed(/^(RatingBar|Label)/, {
+@themed(/^(RatingBar|Label|Legend)/, {
```

Comment:
I'm not sure why react-ui uses `Label` here, because generally we try to keep each component with unique classnames like `RatingBar_Label` (perhaps they are using `Label` as a standard class across many of the components.

The combination of Label and Legend is a little ambiguous to me. Maybe `RatingBar_Prefix` would be a better name for that class.